saleor/csv/__init__.py

```diff
@@ -17,3 +17,13 @@ class ExportEvents:
             "Email with link to download csv file was sent to the customer.",
         ),
     ]
+
+
+class FileTypes:
+    CSV = "csv"
+    XLSX = "xlsx"
+
+    CHOICES = [
+        (CSV, "Plain csv file."),
```

Comment:
```suggestion
        (CSV, "Plain .csv file."),
```
:police_officer:

Reply:
Maybe instead `.csv` it should be uppercase because it's acronym.

Reply:
I think uppercase is a good solution but it need to be changed in both choices.